### ISSUE TITLE
Weird Output When Executing Model

### ISSUE THREAD

**Issue by Nielsien** [adamtiger/NNSharp#8]
Hi, I got a weird output from my trained model in C# when I predict with  normal variables that would work in a Python application.
For example when I use: 
` input = { 30, 5, 198, 12, 37, 0, 0, 0, 2, 94, 1, 0, 30 }`
as input my outcome is 8,6262827794.
When it should be 0.7985896

I used the code from this thread:   https://github.com/adamtiger/NNSharp/issues/3
When I use:

            double[] input = { 30, 5, 198, 12, 37, 0, 0, 0, 2, 94, 1, 0, 30 };
            // Read the previously created json.
            var reader = new ReaderKerasModel("Path_To_Json_Model");
            SequentialModel model = reader.GetSequentialExecutor();

            DataArray data = new DataArray(input.Length);
            for (int i = 0; i < input.Length; i++)
            {
                data[i] = input[i];
            }
            Data2D inputdata = (Data2D)data;

            IData output = model.ExecuteNetwork(inputdata);

            Data2D output2 = (Data2D)output;

            // String representation of the keras model 

            SequentialModelData model2 = (SequentialModelData)model.GetSummary();

            Console.WriteLine(model2.GetStringRepresentation());

            Console.WriteLine(output2[0,0,0,0]);
            
            Console.ReadKey();
            
       
Maybe I did something wrong in the code, but if I look at the architecture of the model I can see that the model is correctly being used.
This is the Json file I used: 
[model.zip](https://github.com/adamtiger/NNSharp/files/2331972/model.zip)

Thanks in advance!
Niels

**Comment by Nielsien** [comment]
I found a mistake in my Neural network, it was that I had more than 1 output node. I changed it so it had 1 output node but the results were not any beter.
Because my output is still a weird number like:
4.108394799

Here's the new model:
[model.zip](https://github.com/adamtiger/NNSharp/files/2332156/model.zip)

**Comment by Nielsien** [comment]
I found the solution, the problem was that my model in Python was normalised and the input in C# not was normalised. After I changed this in Python it worked perfectly.
